neuro_san/http_sidecar/logging/http_logger.py

```diff
@@ -39,6 +40,12 @@ def __init__(self):
         # can only be extracted and used on per-request basis.
         # Async Http server is single-threaded.
         self.logger = logging.getLogger(HttpLogger.HTTP_LOGGER_NAME)
+        # Initialize minimal metadata dictionary to contain some value
+        # for each metadata key we expect to be used.
+        self.base_metadata: Dict[str, Any] = {}
+        for key in forwarded_metadata:
+            self.base_metadata[key] = "None"
+        self.base_metadata["source"] = HttpLogger.HTTP_LOGGER_NAME
 
```

Comment:
Build a "minimal" dictionary for forwarded meta-data keys, so logging record will be initialized for everything we might use.